<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDesign</class>
 <widget class="QMainWindow" name="MainWindowDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>控制程序</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/images.qrc">
    <normaloff>:/images/robot.png</normaloff>:/images/robot.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="Australia"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_statue_text">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#cc0000;&quot;&gt;离线&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_robot_staue_img">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/images.qrc">:/images/disconnect.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>延迟：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Pingms">
        <property name="text">
         <string>~</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_battery" native="true">
        <property name="minimumSize">
         <size>
          <width>76</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>76</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/images.qrc">:/images/power_v.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_power">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>00.00V</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_config">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/images.qrc">
          <normaloff>:/images/config.png</normaloff>:/images/config.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>1800</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tab_manager">
        <property name="minimumSize">
         <size>
          <width>730</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1400</width>
          <height>54531</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Australia"/>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_status">
         <attribute name="title">
          <string>视频</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="1" column="1">
              <widget class="QLabel" name="label_video2">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>65536</width>
                 <height>65536</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:1px solid rgb(0, 0, 0)</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../resources/images.qrc">:/images/noVedio.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_video3">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>65536</width>
                 <height>65536</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:1px solid rgb(0, 0, 0)</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../resources/images.qrc">:/images/noVedio.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_video0">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>655536</width>
                 <height>65536</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:1px solid rgb(0, 0, 0)</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../resources/images.qrc">:/images/noVedio.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_video1">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>65536</width>
                 <height>65536</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:1px solid rgb(0, 0, 0)</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../resources/images.qrc">:/images/noVedio.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>导航</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_14" stretch="1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_build_map">
              <item>
               <widget class="QLabel" name="label_rvizShow">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="text">
                 <string>当前未连接</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="move_camera_btn">
              <property name="text">
               <string>Move Camera</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/images.qrc">
                <normaloff>:/images/rotate_cam.svg</normaloff>:/images/rotate_cam.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="set_select_btn">
              <property name="text">
               <string>Select</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/images.qrc">
                <normaloff>:/images/classes/Select.png</normaloff>:/images/classes/Select.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="set_pos_btn">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>2D Pose Estimate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="set_goal_btn">
              <property name="text">
               <string>2D Nav Goal</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="set_3dgoal_btn">
              <property name="text">
               <string>3D Nav Goal</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="set_return_btn">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>设置返航点</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/images.qrc">
                <normaloff>:/images/set_return.png</normaloff>:/images/set_return.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="return_btn">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>返航</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/images.qrc">
                <normaloff>:/images/return.png</normaloff>:/images/return.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>模型</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_robotmodel">
            <item>
             <widget class="QLabel" name="label_modelShow">
              <property name="text">
               <string>当前未连接</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>1800</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>导航</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_001" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_view">
                  <property name="minimumSize">
                   <size>
                    <width>63</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>导航视角:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_setView">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>第三人称</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_floor">
               <property name="text">
                <string>上楼</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_floor_2">
               <property name="text">
                <string>下楼</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>462</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;功能</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowDesign</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
